src/system/scrub.rs

```diff
@@ -67,15 +68,40 @@ where
 ///
 /// Contains information about the amount of data processed by the scrubber (in bytes),
 /// time spent on scrubbing,
-/// and the amount of data left untouched.
-#[derive(Debug, Default, PartialEq, Eq, Copy, Clone)]
+/// the amount of data left untouched and clustering.
```

Comment:
```suggestion
/// the amount of data left untouched and clustering information.
```
не совсем ясно что за кластеринг такой